### DISCUSSION TITLE
multiple birdseye views

### DISCUSSION THREAD

**Discussion by drocanyd** [blakeblackshear/frigate#3394] [Ideas]
I think it would be beneficial to a lot of users to have a way to have multiple birdseye views, with configurable camera selection. 

If one could go to a webpage that would be birdseye by default like it already is, or birdseye2, birdseye3, and so on for other views.  Each view should be able to specify what cameras are displayed. 

this would allow for different configurations on different devices/areas in the home.

**Comment by toxic0berliner** [comment]
I would really like that too!
My need for it has lessen a bit with the new beta since it will at least start removing old recordings when the drive is full but in fact it's still far from ideal.
I would still prefer having several birdseye views, have my Synology record them and handle archiving, I can setup a cap in days old for the footage but also in gigabytes used, the first rule to match deletes the oldest footage. 
Anyway this is also a very good way to reduce the quality but keep archives for longer.
As well as a great way to show many cameras at once on HomeAssistant dashboards for example...
